pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -83,8 +108,34 @@ func TestGolden(t *testing.T) {
 		{
 			base: "stats_inclusion",
 			annotations: map[string]string{
-				"sidecar.istio.io/statsInclusionPrefixes": "cluster_manager,cluster.xds-grpc,listener.",
+				"sidecar.istio.io/statsInclusionPrefixes": "prefix1,prefix2",
+				"sidecar.istio.io/statsInclusionSuffixes": "suffix1,suffix2",
 			},
+			stats: stats{prefixes: "prefix1,prefix2",
+				suffixes: "suffix1,suffix2"},
+		},
+		{
+			base: "stats_inclusion",
```

Comment:
Ed these are additional tests that test the new annotations.
I did not add more golden files, but did tests in a different way.